packages/kbn-search-index-documents/components/result/rich_result_header.tsx

```diff
@@ -60,6 +61,7 @@ const MetadataPopover: React.FC<MetaDataProps> = ({
   onDocumentDelete,
   score,
   showScore = false,
+  hasDeleteDocumentsPrivilege = undefined,
```

Comment:
nitpick: hasDeleteDocumentsPrivilege has undefined by default already if you not pass it.

Reply:
updated in aaccd0bfe963c5be0817e64cbc28edfc2fad4193